common-lib/src/main/scala/models/Status.scala

```diff
@@ -19,11 +19,12 @@ object Status {
   }
   def Stub = Status("Stub")
   def Writers = Status("Writers")
-  def Subs = Status("Subs")
+  def Desk = Status("Desk")
```

Comment:
Is there any particular reason why is there a mix between using `Status.Writers` and `Status("Writers")`?